cmd/crc/cmd/start.go

```diff
@@ -58,6 +59,7 @@ func runStart(arguments []string) {
 		NameServer:    crcConfig.GetString(config.NameServer.Name),
 		GetPullSecret: getPullSecretFileContent,
 		Debug:         isDebugLog(),
+		Proxy:         handleProxyConfig,
```

Comment:
I think you can pass directly the `ProxyConfig` here rather than a callback? Nothing is using this `Proxy` field though?

Reply:
@cfergeau right now yes, nothing is using it but it should be used as soon as we start the cluster proxy part so I just put the place holder before.

This callback will allow me to return the error in case of fail to validate the proxy URI.

Reply:
> @cfergeau right now yes, nothing is using it but it should be used as soon as we start the cluster proxy part so I just put the place holder before.

I'd prefer that we add this when it starts getting used. Maybe during review of the cluster part, we'll realize things need to be done differently.

> This callback will allow me to return the error in case of fail to validate the proxy URI.

I'd prefer if we could make this simpler.. ;)